url/query/params_test.go

```diff
@@ -0,0 +1,62 @@
+package query_test
+
+import (
+	"net/url"
+	"reflect"
+	"testing"
+
+	. "github.com/microdevs/missy/url/query"
+)
+
+var testParams = []struct {
+	in  string
+	out Values
+}{
+	{"http://localhost?value=gt:10", Values{"value": Operators{GT: []string{"10"}}}},
+	{"http://localhost?value=gt:10&value=lte:100", Values{"value": Operators{GT: []string{"10"}, LTE: []string{"100"}}}},
+	{"http://localhost?sort=asc:key1", Values{"sort": Operators{Asc: []string{"key1"}}}},
+	{"http://localhost?sort=desc:key1,key2,key3", Values{"sort": Operators{Desc: []string{"key1"}, "": []string{"key2", "key3"}}}},
+	{"http://localhost?sort=desc:key1,key2,asc:key3", Values{"sort": Operators{Asc: []string{"key3"}, Desc: []string{"key1"}, "": []string{"key2"}}}},
+	{"http://localhost?date=gt:2018-01-01&date=lte:2018-04-01", Values{"date": Operators{GT: []string{"2018-01-01"}, LTE: []string{"2018-04-01"}}}},
+	{"http://localhost?name=name&first_value=100&desc_value=inner", Values{"name": Operators{"": []string{"name"}}, "first_value": Operators{"": []string{"100"}}, "desc_value": Operators{"": []string{"inner"}}}},
```

Comment:
Can we add a `Default = ""` const for those? Would be easier to understand for new developers.

Reply:
sure

Reply:
awesome, thanks 🙂